🌐 Add Korean translation for `docs/benchmarks.md`

kimjaeyoonn opened the pull request:
I'm posting a new PR because there are a few things that need to be corrected through feedback.
Thank you 😆

Add Korean translation for ko/docs/benchmarks.md
Related [FEATURE] Korean translations [FEATURE] Korean translations [FEATURE] Korean translations #2017

kimjaeyoonn committed:
🌐 Add Korean translation for docs/benchmarks.md

kimjaeyoonn committed:
Update mkdocs.yml

DongwonGim reviewed (commented):
Starlette 및 Uvicorn보다 낮습니다.
->
Starlette과 Uvicorn 다음으로 높은 순위입니다.

0417taehyun commented on docs/ko/docs/benchmarks.md:
```diff
@@ -0,0 +1,34 @@
+# 벤치마크
+
+독립적인 TechEmpower 벤치마크에 따르면 Uvicorn에서 실행되는 **FastAPI** 애플리케이션을 <a href="https://www.techempower.com/benchmarks/#section=test&runid=7464e520-0dc2-473d-bd34-dbdfd7e85911&hw=ph&test=query&l=로 표시합니다. zijzen-7" class="external-link" target="_blank">사용 가능한 가장 빠른 Python 프레임워크 중 하나</a>로  Starlette 및 Uvicorn보다 낮습니다. (FastAPI에서 내부적으로 사용). (*)
```
```suggestion
독립적인 TechEmpower 벤치마크에 따르면 Uvicorn에서 실행되는 **FastAPI** 응용 프로그램을 <a href="https://www.techempower.com/benchmarks/#section=test&runid=7464e520-0dc2-473d-bd34-dbdfd7e85911&hw=ph&test=query&l=zijzen-7" class="external-link" target="_blank">사용 가능한 가장 빠른 파이썬 프레임워크 중 하나</a>로  표시하며, (FastAPI에서 내부적으로 사용한) Starlette 및 Uvicorn 다음으로 높은 순위입니다.(*)
```

@DongwonGim 
Using `suggestion` is better solution to check the review point.

0417taehyun commented on docs/ko/docs/benchmarks.md:
```diff
@@ -0,0 +1,34 @@
+* **Uvicorn**:
+  * 서버 자체를 제외하고 추가 코드가 없기 때문에 최고의 성능을 제공합니다.
+  * Uvicorn에서 직접 응용 프로그램을 작성하지 않습니다. 즉, 사용자의 코드에는 적어도 Starlette(또는 **FastAPI**)에서 제공하는 모든 코드가 포함되어야 합니다. 그렇게 하면 최종 애플리케이션은 프레임워크를 사용하고 앱 코드와 버그를 최소화하는 것과 동일한 오버헤드를 갖게 됩니다.
+  * Uvicorn을 비교하고 싶다면 Daphne, Hypercorn, uWSGI 등의 Application Server와 비교하는 것이 좋습니다.
```
```suggestion
  * Uvicorn을 비교하고 싶다면 Daphne, Hypercorn, uWSGI 등의 응용 프로그램 서버와 비교하는 것이 좋습니다.
```

> Also, use less English as you can.

by #2017 .

0417taehyun commented on docs/ko/docs/benchmarks.md:
```diff
@@ -0,0 +1,34 @@
+  * Uvicorn을 비교하고 싶다면 Daphne, Hypercorn, uWSGI 등의 Application Server와 비교하는 것이 좋습니다.
+* **Starlette**:
+  * Uvicorn 다음으로 최고의 성능을 보입니다. 실제로 Starlette는 Uvicorn을 사용하여 실행합니다. 따라서 더 많은 코드를 실행해야 하기 때문에 Uvicorn보다 느릴 수 있습니다.
+  * 그러나 경로 등을 기반으로 하는 라우팅을 사용하여 간단한 웹 응용 프로그램을 빌드할 수 있는 도구를 제공합니다.
```
```suggestion
  * 그러나 경로 등을 기반으로 하는 라우팅을 사용하여 간단한 웹 응용 프로그램을 구축할 수 있는 도구를 제공합니다.
```

> Also, use less English as you can.

by #2017 .

0417taehyun commented on docs/ko/docs/benchmarks.md:
```diff
@@ -0,0 +1,34 @@
+  * Starlette을 비교하고 싶다면 Sanic, Flask, Django 등의 웹 프레임워크(또는 마이크로프레임워크)와 비교하는 것이 좋습니다.
+* **FastAPI**:
+  * Starlette이 Uvicorn을 사용하여 빠른 속도를 제공하는 것처럼 **FastAPI**는 Starlette을 사용하므로 무척 빠른 속도를 제공합니다.
+  * FastAPI는 Starlette에 추가적으로 더 많은 기능을 제공합니다. 데이터 유효성 검사 및 직렬화와 같이 API를 구축할 때 거의 매번 필요한 기능입니다. 그리고 그것을 사용하면 무료로 자동 문서를 얻을 수 있습니다(자동 문서는 실행 중인 응용 프로그램에 오버헤드를 추가하지 않으며 시작 시 생성됩니다).
```
```suggestion
  * FastAPI는 Starlette에 추가적으로 더 많은 기능을 제공합니다. 그 기능들은 데이터 검증 및 직렬화와 같이 API를 구축할 때 거의 매번 필요한 것들 입니다. 그리고 그것들을 사용하면 무료로 자동 문서를 얻을 수 있습니다 (자동 문서는 실행 중인 응용 프로그램에 오버헤드를 추가하지 않으며 시작 시 생성됩니다).
```

0417taehyun commented on docs/ko/docs/benchmarks.md:
```diff
@@ -0,0 +1,34 @@
+* **FastAPI**:
+  * Starlette이 Uvicorn을 사용하여 빠른 속도를 제공하는 것처럼 **FastAPI**는 Starlette을 사용하므로 무척 빠른 속도를 제공합니다.
+  * FastAPI는 Starlette에 추가적으로 더 많은 기능을 제공합니다. 데이터 유효성 검사 및 직렬화와 같이 API를 구축할 때 거의 매번 필요한 기능입니다. 그리고 그것을 사용하면 무료로 자동 문서를 얻을 수 있습니다(자동 문서는 실행 중인 응용 프로그램에 오버헤드를 추가하지 않으며 시작 시 생성됩니다).
+  * FastAPI를 사용하지 않고 Starlette를 직접 사용한 경우(혹은 Sanic, Flask, Responder 등과 같은 다른 도구 사용)에는 모든 데이터 유효성 검사 및 직렬화를 직접 구현해야 합니다. 따라서 최종 애플리케이션은 FastAPI를 사용하여 빌드된 것과 동일한 오버헤드를 갖게 됩니다. 그리고 대부분의 경우, 데이터 유효성 검사 및 직렬화는 애플리케이션에서 작성된 코드 중 가장 많은 양입니다.
```
```suggestion
  * FastAPI를 사용하지 않고 Starlette (또는 Sanic, Flask, Responder 등과 같은 다른 도구) 를 직접 사용한 경우 모든 데이터 검증 및 직렬화를 직접 구현해야 합니다. 따라서 최종 응용 프로그램은 FastAPI를 사용하여 구축된 것과 동일한 오버헤드를 갖게 됩니다. 그리고 대부분의 경우, 데이터 검증 및 직렬화는 응용 프로그램에서 작성된 코드 중 가장 많은 양을 차지합니다.
```

0417taehyun commented on docs/ko/docs/benchmarks.md:
```diff
@@ -0,0 +1,34 @@
+  * FastAPI는 Starlette에 추가적으로 더 많은 기능을 제공합니다. 데이터 유효성 검사 및 직렬화와 같이 API를 구축할 때 거의 매번 필요한 기능입니다. 그리고 그것을 사용하면 무료로 자동 문서를 얻을 수 있습니다(자동 문서는 실행 중인 응용 프로그램에 오버헤드를 추가하지 않으며 시작 시 생성됩니다).
+  * FastAPI를 사용하지 않고 Starlette를 직접 사용한 경우(혹은 Sanic, Flask, Responder 등과 같은 다른 도구 사용)에는 모든 데이터 유효성 검사 및 직렬화를 직접 구현해야 합니다. 따라서 최종 애플리케이션은 FastAPI를 사용하여 빌드된 것과 동일한 오버헤드를 갖게 됩니다. 그리고 대부분의 경우, 데이터 유효성 검사 및 직렬화는 애플리케이션에서 작성된 코드 중 가장 많은 양입니다.
+  * 따라서 FastAPI를 사용하면 개발 시간, 버그, 코드 양을 절약할 수 있으며, 사용하지 않는 경우와 동일하거나 더 나은 성능을 얻을 수 있습니다.
+  * FastAPI를 비교하고 싶다면 Flask-apispec, NestJS, Molten 등과 같은 데이터 유효성 검사, 직렬화 및 문서화를 제공하는 웹 애플리케이션 프레임워크(또는 도구 세트)와 비교하는 것이 좋습니다.
```
```suggestion
  * FastAPI를 비교하고 싶다면 Flask-apispec, NestJS, Molten 등과 같은 데이터 검증, 직렬화 및 문서화를 제공하는 웹 응용 프로그램 프레임워크 (또는 도구 집합) 와 비교하는 것이 좋습니다.
```

> Also, use less English as you can.

by #2017 .

0417taehyun commented on docs/ko/docs/benchmarks.md:
```diff
@@ -0,0 +1,34 @@
+
+벤치마크를 확인할 때 서로 다른 유형의 여러 도구를 동등하게 비교하는 것이 일반적입니다.
+
+특히 Uvicorn, Starlette 및 FastAPI를 함께 비교하는 것을 확인하기 위해서(다른 많은 도구 중에서).
```
```suggestion
특히 (다른 많은 도구 중에서) Uvicorn, Starlette 및 FastAPI를 함께 비교하는 것을 확인하기 위한 방법이 필요합니다.
```

I think it would be better to complete the sentence by the context. The next sentence refers to solution of the problem of comparing several tools. So it the previous sentence-this sentence-might be completed like `방법이 필요합니다.`

0417taehyun commented on docs/ko/docs/benchmarks.md:
```diff
@@ -0,0 +1,34 @@
+
+* **Uvicorn**:
+  * 서버 자체를 제외하고 추가 코드가 없기 때문에 최고의 성능을 제공합니다.
+  * Uvicorn에서 직접 응용 프로그램을 작성하지 않습니다. 즉, 사용자의 코드에는 적어도 Starlette(또는 **FastAPI**)에서 제공하는 모든 코드가 포함되어야 합니다. 그렇게 하면 최종 애플리케이션은 프레임워크를 사용하고 앱 코드와 버그를 최소화하는 것과 동일한 오버헤드를 갖게 됩니다.
```
```suggestion
  * Uvicorn에서 직접 응용 프로그램을 작성하지 않습니다. 즉, 사용자의 코드에는 적어도 Starlette(또는 **FastAPI**)에서 제공하는 모든 코드가 포함되어야 합니다. 그렇게 하면 최종 응용 프로그램은 프레임워크를 사용하고 앱 코드와 버그를 최소화하는 것과 동일한 오버헤드를 갖게 됩니다.
```

0417taehyun reviewed (changes requested):
This review is my mistake 😭

kimjaeyoonn committed:
Update docs/ko/docs/benchmarks.md

Co-authored-by: weekwith.me <63915557+0417taehyun@users.noreply.github.com>

kimjaeyoonn committed:
Update docs/ko/docs/benchmarks.md

Co-authored-by: weekwith.me <63915557+0417taehyun@users.noreply.github.com>

kimjaeyoonn committed:
Update docs/ko/docs/benchmarks.md

Co-authored-by: weekwith.me <63915557+0417taehyun@users.noreply.github.com>

kimjaeyoonn committed:
Update docs/ko/docs/benchmarks.md

Co-authored-by: weekwith.me <63915557+0417taehyun@users.noreply.github.com>

kimjaeyoonn committed:
Update docs/ko/docs/benchmarks.md

Co-authored-by: weekwith.me <63915557+0417taehyun@users.noreply.github.com>

kimjaeyoonn committed:
Update docs/ko/docs/benchmarks.md

Co-authored-by: weekwith.me <63915557+0417taehyun@users.noreply.github.com>

kimjaeyoonn committed:
Update docs/ko/docs/benchmarks.md

Co-authored-by: weekwith.me <63915557+0417taehyun@users.noreply.github.com>

kimjaeyoonn commented:
I have accepted the feedback. Thank you @0417taehyun  😆
And I'm sure there are no more problems with the translation of this document.

@tiangolo I think you can merge this PR :)

kimjaeyoonn committed:
Update benchmarks.md

@DongwonGim    
I have accepted your feedback. Thankyou 🚀

@tiangolo
I hope you merge this document.
There are many documents still open in #2017. 
I'm curious as to why it can't be merged. 
Even if it's not perfect, if you merge it, it will be easier for people to find parts to fix on the FastAPI webpage, so please merge it.

kimjaeyoonn commented:
@DongwonGim    
I have accepted your feedback. Thankyou 🚀

@tiangolo
I hope you merge this document.
There are many documents still open in #2017. 
I'm curious as to why it can't be merged. 
Even if it's not perfect, if you merge it, it will be easier for people to find parts to fix on the FastAPI webpage, so please merge it.

DevDae reviewed (approved):
I review all works and check there is nothing to revise! 
Appreciate you, @kimjaeyoonn 🚀

tiangolo commented:
Thanks for the contribution @kimjaeyoonn! As I don't speak the language, I merge translation PRs after they have 2 approving reviews. 🤓
